webdemo/collation/index.html

```diff
@@ -66,19 +66,20 @@ <h1><a href="https://icu.unicode.org/">ICU</a> Collation Demo</h1>
 <option value="en_US_POSIX">en-US-u-va-posix (type=standard):   American English (Computer, Standard Sort Order)</option>
 <option value="eo">eo (type=standard):   Esperanto (Standard Sort Order)</option>
 <option value="es">es (type=standard):   Spanish (Standard Sort Order)</option>
-<option value="es@collation=search">es-u-co-search (normalization=on):   Spanish (General-Purpose Search)</option>
+<option value="es@collation=search">es-u-co-search (normalization=on):   Spanish (General-Purpose Search)</option> meld webdemo/collation/index.html ../available-collators.txt
```

Comment:
```suggestion
<option value="es@collation=search">es-u-co-search (normalization=on):   Spanish (General-Purpose Search)</option>
```

looks like a "paste" gone astray

Reply:
Done. Thanks for the careful review. PTAL.